frontend/src/scenes/data-management/ingestion-warnings/IngestionWarningsView.tsx

```diff
@@ -126,6 +127,19 @@ const WARNING_TYPE_RENDERER = {
             </>
         )
     },
+    message_size_too_large: function Render(warning: IngestionWarning): JSX.Element {
+        const details = warning.details as {
+            eventUuid: string
+            distinctId: string
+        }
+        return (
+            <>
+                Discarded event for distinct_id{' '}
+                <Link to={urls.personByDistinctId(details.distinctId)}>{details.distinctId}</Link> that exceeded 1MB in
+                size after processing: <code>{details.eventUuid}</code>
```

Comment:
nit: it's a bit confusing what's at the end, that it's the event UUID, but feel free to ignore

Reply:
Yeah, I'm not sure how useful it is to include here either. I could remove it, or I could more clearly denote it, like these ones:

https://github.com/PostHog/posthog/blob/0aa736ca581263f24557bbffda5d30c9d45c4926/frontend/src/scenes/data-management/ingestion-warnings/IngestionWarningsView.tsx#L47

https://github.com/PostHog/posthog/blob/0aa736ca581263f24557bbffda5d30c9d45c4926/frontend/src/scenes/data-management/ingestion-warnings/IngestionWarningsView.tsx#L62

For example
```suggestion
                Discarded event for distinct_id{' '}
                <Link to={urls.personByDistinctId(details.distinctId)}>{details.distinctId}</Link> that exceeded 1MB in
                size after processing (event uuid: <code>{details.eventUuid}</code>)
```

Any thoughts/preferences? I don't have a strong opinion on this.

Reply:
No strong opinion here either, but making it like the existing ones seems best.